src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -64,15 +68,15 @@
 
 (defn view
   []
-  (let [[selected-tab set-selected-tab] (rn/use-state (:id (first tabs-data)))
-        account-list-ref                (rn/use-ref-atom nil)
-        tokens-loading?                 (rf/sub [:wallet/home-tokens-loading?])
-        networks                        (rf/sub [:wallet/selected-network-details])
-        account-cards-data              (rf/sub [:wallet/account-cards-data])
-        cards                           (conj account-cards-data (new-account-card-data))
-        [init-loaded? set-init-loaded]  (rn/use-state false)
-        {:keys [formatted-balance]}     (rf/sub [:wallet/aggregated-token-values-and-balance])
-        theme                           (quo.theme/use-theme)]
+  (let [selected-tab                   (or (rf/sub [:wallet/home-tab]) first-tab-id)
```

Comment:
The expression:

 `(or (rf/sub[...]) value-if-not-found)`

Seems not necessary, since this is a new sub and it'll be cleaner if all the logic is wrapped within the sub.

We have a `def` for `first-tab-id`, instead of putting it in the view, we can define it near the subscription, so the sub directly returns.

Wdyt?